scripts/dandi_uploads.py

```diff
@@ -73,3 +81,6 @@ def set_env():
     print("STATUS")
     print(list(status))
     os.remove(args.dandi_file)
+    
+    if args.final == 'True':
+        fb.update_session_dandi(args.project_id, args.session_id, "sub_" + args.subject_id + '/' + args.sess_id)
```

Comment:
So you assign the link based on your storage convention on dandi. Is that convention set at upload? So we are creating a dependency between two repositories. Let's discuss at our meeting today. Just wanted to understand your vision.

Reply:
So we discuss and the front end only pull which is good.